_pages/bea/bea18.md

```diff
@@ -44,3 +44,81 @@ toc_icon: 'cog'
+
+## Submission Information
+
+We will be using the ACL Submission Guidelines for the BEA Workshop this year. Authors are invited to submit a long paper of up to eight (8) pages of content, plus unlimited references; final versions of long papers will be given one additional page of content (up to 9 pages) so that reviewers’ comments can be taken into account. We also invite short papers of up to four (4) pages of content, plus unlimited references. Upon acceptance, short papers will be given five (5) content pages in the proceedings. Authors are encouraged to use this additional page to address reviewers’ comments in their final versions. Papers which describe systems are also invited to give a demo of their system. If you would like to present a demo in addition to presenting the paper, please make sure to select either “long paper + demo” or “short paper + demo” under “Submission Category” in the START submission page.
+
+Previously published papers cannot be accepted. The submissions will be reviewed by the program committee. As reviewing will be blind, please ensure that papers are anonymous. Self-references that reveal the author’s identity, e.g., “We previously showed (Smith, 1991) …”, should be avoided. Instead, use citations such as “Smith previously showed (Smith, 1991) …”.
+
+We have also included conflict of interest in the submission form. You should mark all potential reviewers who have been authors on the paper, are from the same research group or institution, or who have seen versions of this paper or discussed it with you.
+
+We will be using the START conference system to manage submissions: [https://www.softconf.com/acl2023/bea2023/](https://www.softconf.com/acl2023/bea2023/)
+
+### LaTeX and Word Templates
+
+Paper submissions must use the official ACL style templates, which are available from [here](https://github.com/acl-org/acl-style-files). Please follow the paper formatting guidelines general to "\*ACL" conferences available [here](https://acl-org.github.io/ACLPUB/formatting.html). Authors may not modify these style files or use templates designed for other conferences. Submissions that do not conform to the required styles, including paper size, margin width, and font size restrictions, will be rejected without review.
+
+### Double Submission Policy
+
+We will follow the official ACL double-submission policy. Specifically, papers being submitted both to BEA and another conference or workshop must:
+- Note on the title page the other conference or workshop to which they are being submitted.
+- State on the title page that if the authors choose to present their paper at BEA (assuming it was accepted), then the paper will be withdrawn from other conferences and workshops.
```

Comment:
Organizing Committee
- [Ekaterina Kochmar](https://ekochmar.github.io/about/), University of Bath
- [Jill Burstein](https://sites.google.com/site/jbursteinets/), Duolingo
- [Andrea Horbach](https://www.fernuni-hagen.de/forschung/schwerpunkte/d2l2/team/andrea.horbach.shtml), FernUniversität in Hagen
- [Ronja Laarmann-Quante](https://www.ltl.uni-due.de/team/ronja-laarmann-quante), University of Duisburg-Essen
- [Nitin Madnani](https://desilinguist.org/), Educational Testing Service
- [Anaïs Tack](https://anaistack.github.io/), Stanford University
- [Victoria Yaneva](http://www.victoriayaneva.info/), National Board of Medical Examiners
- [Zheng Yuan](https://www.cl.cam.ac.uk/~zy249/), University of Cambridge
- [Torsten Zesch](https://www.ltl.uni-due.de/team/torsten-zesch), FernUniversität in Hagen

Workshop contact email address: [bea.nlp.workshop@gmail.com](mailto:bea.nlp.workshop@gmail.com)

Reply:
### Program Committee

- Rania Abdelghani, INRIA Bordeaux - EvidenceB Paris
- Tazin Afrin, Educational Testing Services
- David Alfter, Université catholique de Louvain
- Erfan Al-Hossami, University of North Carolina at Charlotte
- Michael Gringo Angelo Bayona, Trinity College Dublin
- Lee Becker, Pearson
- Marie Bexte, FernUniversität in Hagen
- Serge Bibauw, Universidad Central del Ecuador; KU Leuven
- Daniel Brenner, ETS
- Dominique Brunato, Institute of Computational Linguistics / CNR (Pisa, Italy)
- Christopher Bryant, University of Cambridge
- Dumitru-Clementin Cercel, University Politehnica of Bucharest
- Li-Hsin Chang, University of Turku
- Guanliang Chen, Monash University
- Hyundong Cho, USC ISI
- Martin Chodorow, The City University of New York
- Mark Core, University of Southern California
- Kordula De Kuthy, University of Tübingen
- Jasper Degraeuwe, Ghent University
- Carrie Demmans Epp, University of Alberta
- Yuning Ding, FernUniversität in Hagen
- Rahul Divekar, ETS
- George Duenas, Universidad Pedagogica Nacional
- Mariano Felice, British Council
- Michael Flor, Educational Testing Service
- Yo Ehara, Tokyo Gakugei University
- Ananya Ganesh, University of Colorado Boulder
- Lingyu Gao, TTIC
- Samuel González-López, Technological University of Nogales Sonora 
- Tanja Heck, University of Tübingen
- Nicolas Hernandez, Nantes University - LS2N
- Chung-Chi Huang, Frostburg State University
- Yi-Ting Huang, NTUST
- Joseph Marvin Imperial, University of Bath
- Radu Tudor Ionescu, University of Bucharest
- Qinjin Jia, North Carolina State University
- Richard Johansson, University of Gothenburg
- Elma Kerz, RWTH Aachen University
- Mamoru Komachi, Tokyo Metropolitan University
- Kristopher Kyle, University of Oregon
- Ji-Ung Lee, UKP Lab, TU Darmstadt, Germany
- Arun Balajiee Lekshmi Narayanan, University of Pittsburgh
- Xu Li, Zhejiang University
- Diane Litman, University of Pittsburgh
- Yudong Liu, Western Washington University
- Zhexiong Liu, University of Pittsburgh 
- Zoey Liu, University of Florida
- Peter Ljunglöf, University of Gothenburg
- Anastassia Loukina, Grammarly Inc.
- Gunnar Lund, Grammarly
- Jakub Macina, ETH Zurich
- Lieve Macken, Ghent University, Belgium
- James Martin, University of Colorado Boulder
- Sandeep Mathias, Presidency University, Bangalore
- Janet Mee, NBME
- Detmar Meurers, University of Tübingen
- Farah Nadeem, UNICEF
- Ben Naismith, Duolingo
- Sungjin Nam, ACT Inc.
- Diane Napolitano, Associated Press
- Kamel Nebhi, Education First
- Hwee Tou Ng, National University of Singapore
- Huy Nguyen, Amazon
- Kostiantyn Omelianchuk, Grammarly
- Simon Ostermann, Deutsches Forschungszentrum für Künstliche Intelligenz
- Robert Östling, Department of Linguistics, Stockholm University
- Ulrike Pado, Hochschule fuer Technik Stuttgart
- Lis Pereira, Ochanomizu University
- Reinald Adrian Pugoy, University of the Philippines
- Long Qin, Alibaba Cloud
- Mengyang Qiu, University at Buffalo
- Martí Quixal, University of Tübingen
- Vipul Raheja, Grammarly
- Vivi Rantung, Universitas Negeri Manado
- Arjun Ranesh Rao, Microsoft
- Manav Rathod, Glean
- Brian Riordan, ETS
- Frankie Robertson, University of Jyväskylä
- Aiala Rosá, Instituto de Computación, Facultad de Ingeniería, Udelar
- Alla Rozovskaya, Queens College, CUNY
- Katherine Stasaski, Salesforce Research
- Helmer Strik, Radboud University
- Jan Švec, University of West Bohemia
- Anaïs Tack, KU Leuven, UCLouvain
- Elaine Uí Dhonnchadha, Trinity College Dublin
- Alexandra Uitdenbogerd, RMIT
- Sowmya Vajjala, National Research Council, Canada
- Giulia Venturi, Institute for Computational Linguistics "A. Zampolli" (CNR-ILC)
- Carl Vogel, Trinity College Dublin
- Elena Volodina, University of Gothenburg, Sweden
- Michael White, The Ohio State University
- Alistair Willis, The Open University
- Yiqiao Xu, North Carolina State University
- Klaus Zechner, ETS
- Torsten Zesch, FernUniversität in Hagen